src/internal.c

```diff
@@ -7977,15 +8103,45 @@ int SendKexDhInit(WOLFSSH* ssh)
 #else
             ret = WS_INVALID_ALGO_ID;
 #endif /* !defined(WOLFSSH_NO_ECDH) */
+#ifndef WOLFSSH_NO_SABER_LEVEL1_SHA256
+        } else if (ssh->handshake->useSaber) {
+            OQS_KEM* kem = NULL;
+            ret = 0;
+
+            if (ret == 0) {
```

Comment:
`ret` is always 0 here...

Reply:
But it looks so nice and consistent.  Anyways, fixed.  Stay tuned.